src/utilities/convert-nodes.ts

```diff
@@ -44,10 +57,9 @@ export const convertToInternalNode = (node: NodeProps): InternalNode => {
     data: {
       title,
       disabled,
-      visibleFields: getFieldsWithExpandStatus(fields),
+      fields: getInternalFields(fields),
       borderVariant,
       variant,
-      externalNode: node,
```

Comment:
could this have been where we passed ReactNode to the package and the diagram disappeared? 🤔